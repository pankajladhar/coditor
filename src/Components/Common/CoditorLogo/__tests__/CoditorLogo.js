import React from "react";
import renderer from "react-test-renderer";
import CoditorLogo from "./../CoditorLogo";

describe("<CoditorLogo /> ", () => {
  it("should render properly", () => {
    const tree = renderer.create(<CoditorLogo />);
    expect(tree.toJSON()).toMatchSnapshot();
  });

  it("should render properly when name is passed", () => {
    const tree = renderer.create(<CoditorLogo name />);
    expect(tree.toJSON()).toMatchSnapshot();
  });

  it("should render properly when size and animate property passed", () => {
    const tree = renderer.create(<CoditorLogo size={200} animate/>);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
