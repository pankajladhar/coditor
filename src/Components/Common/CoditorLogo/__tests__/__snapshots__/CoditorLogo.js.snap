// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CoditorLogo />  should render properly 1`] = `
<div
  className="CoditorLogo"
>
  <svg
    fill="none"
    height={100}
    viewBox="0 0 353 353"
    width={100}
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0)"
    >
      <g
        filter="url(#filter0_d)"
        id=""
      >
        <rect
          height="226"
          rx="57"
          stroke="url(#paint0_linear)"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="22"
          transform="rotate(53.2373 199.658 18.6041)"
          width="226"
          x="199.658"
          y="18.6041"
        />
      </g>
      <rect
        fill="url(#paint1_linear)"
        height="165.322"
        rx="35"
        transform="rotate(53.2373 193.443 61)"
        width="165.322"
        x="193.443"
        y="61"
      />
      <path
        d="M138.755 211.597C139.575 211.597 139.985 212.622 139.985 214.673C139.985 216.753 139.575 217.793 138.755 217.793H131.548C127.475 217.793 124.531 216.826 122.715 214.892C120.781 212.812 119.814 209.238 119.814 204.17V190.063C119.814 187.837 119.565 186.416 119.067 185.801C118.276 184.834 116.841 184.204 114.761 183.911C113.911 183.794 113.486 182.739 113.486 180.747C113.486 178.96 113.911 177.978 114.761 177.803C116.811 177.51 118.086 177.041 118.584 176.396C119.404 175.371 119.814 173.774 119.814 171.606V151.348C119.814 143.994 122.012 139.57 126.406 138.076C127.871 137.578 129.585 137.329 131.548 137.329H138.755C139.575 137.329 139.985 138.34 139.985 140.361C139.985 142.471 139.575 143.525 138.755 143.525H135.283C132.353 143.525 130.42 144.228 129.482 145.635C128.75 146.719 128.384 148.711 128.384 151.611V171.167C128.384 174.8 126.963 177.51 124.121 179.297C123.125 179.941 121.909 180.439 120.474 180.791C125.747 182.314 128.384 185.61 128.384 190.679V204.258C128.384 206.953 128.662 208.682 129.219 209.443C130.244 210.879 132.266 211.597 135.283 211.597H138.755Z"
        fill="#8BB3D0"
      />
      <path
        d="M190.181 169.277L187.632 171.211C185.084 168.047 181.802 166.465 177.789 166.465C174.302 166.465 171.314 167.71 168.824 170.2C166.363 172.661 165.132 175.635 165.132 179.121C165.132 182.607 166.363 185.596 168.824 188.086C171.314 190.547 174.302 191.777 177.789 191.777C179.752 191.777 181.583 191.352 183.282 190.503C185.01 189.653 186.475 188.481 187.676 186.987L190.137 189.053C188.673 190.898 186.856 192.349 184.688 193.403C182.52 194.429 180.22 194.941 177.789 194.941C173.424 194.941 169.688 193.403 166.583 190.327C163.507 187.222 161.968 183.486 161.968 179.121C161.968 174.756 163.507 171.035 166.583 167.959C169.688 164.853 173.424 163.301 177.789 163.301C180.25 163.301 182.564 163.843 184.732 164.927C186.9 165.981 188.716 167.432 190.181 169.277ZM195.103 165.322L192.598 167.256C190.841 165.029 188.658 163.286 186.05 162.026C183.472 160.766 180.718 160.137 177.789 160.137C172.545 160.137 168.062 161.997 164.341 165.718C160.65 169.438 158.804 173.906 158.804 179.121C158.804 184.336 160.65 188.804 164.341 192.524C168.062 196.245 172.545 198.105 177.789 198.105C180.718 198.105 183.487 197.475 186.094 196.216C188.702 194.927 190.884 193.169 192.642 190.942L195.059 192.964C193.008 195.542 190.474 197.578 187.457 199.072C184.439 200.537 181.216 201.269 177.789 201.269C173.775 201.269 170.069 200.288 166.671 198.325C163.272 196.333 160.577 193.638 158.585 190.239C156.622 186.841 155.64 183.135 155.64 179.121C155.64 175.107 156.622 171.401 158.585 168.003C160.577 164.604 163.272 161.924 166.671 159.961C170.069 157.969 173.775 156.973 177.789 156.973C181.216 156.973 184.439 157.72 187.457 159.214C190.504 160.679 193.052 162.715 195.103 165.322ZM200.069 161.367L197.608 163.345C195.235 160.386 192.32 158.057 188.863 156.357C185.406 154.658 181.715 153.808 177.789 153.808C174.361 153.808 171.08 154.482 167.945 155.83C164.84 157.148 162.144 158.95 159.859 161.235C157.603 163.491 155.801 166.186 154.454 169.321C153.135 172.427 152.476 175.693 152.476 179.121C152.476 182.549 153.135 185.83 154.454 188.965C155.801 192.07 157.603 194.766 159.859 197.051C162.144 199.307 164.84 201.108 167.945 202.456C171.08 203.774 174.361 204.433 177.789 204.433C181.715 204.433 185.406 203.584 188.863 201.885C192.32 200.185 195.235 197.856 197.608 194.897L199.981 196.963C197.315 200.273 194.049 202.881 190.181 204.785C186.314 206.66 182.183 207.598 177.789 207.598C173.922 207.598 170.23 206.85 166.715 205.356C163.199 203.833 160.167 201.811 157.618 199.292C155.098 196.743 153.077 193.711 151.553 190.195C150.059 186.68 149.312 182.988 149.312 179.121C149.312 175.254 150.059 171.562 151.553 168.047C153.077 164.531 155.098 161.514 157.618 158.994C160.167 156.445 163.199 154.424 166.715 152.93C170.23 151.406 173.922 150.644 177.789 150.644C182.213 150.644 186.358 151.597 190.225 153.501C194.122 155.405 197.403 158.027 200.069 161.367ZM204.991 157.412L202.442 159.302C199.483 155.64 195.85 152.754 191.544 150.644C187.266 148.535 182.681 147.48 177.789 147.48C173.511 147.48 169.41 148.315 165.484 149.985C161.588 151.655 158.218 153.911 155.377 156.753C152.564 159.565 150.323 162.934 148.653 166.86C146.983 170.757 146.148 174.844 146.148 179.121C146.148 183.398 146.983 187.5 148.653 191.426C150.323 195.322 152.564 198.691 155.377 201.533C158.218 204.346 161.588 206.587 165.484 208.257C169.41 209.927 173.511 210.762 177.789 210.762C182.681 210.762 187.266 209.707 191.544 207.598C195.85 205.488 199.483 202.602 202.442 198.94L204.947 200.874C201.695 204.917 197.696 208.11 192.95 210.454C188.233 212.768 183.179 213.926 177.789 213.926C173.072 213.926 168.56 213.003 164.254 211.157C159.976 209.311 156.285 206.836 153.179 203.73C150.074 200.625 147.598 196.933 145.753 192.656C143.907 188.349 142.984 183.838 142.984 179.121C142.984 174.404 143.907 169.907 145.753 165.63C147.598 161.323 150.074 157.617 153.179 154.512C156.285 151.406 159.976 148.931 164.254 147.085C168.56 145.239 173.072 144.316 177.789 144.316C183.179 144.316 188.248 145.488 192.994 147.832C197.74 150.146 201.739 153.34 204.991 157.412Z"
        fill="#E9B34B"
      />
      <path
        d="M207.462 140.361C207.462 138.34 207.873 137.329 208.693 137.329H216.076C219.972 137.329 222.843 138.369 224.689 140.449C226.623 142.617 227.604 146.25 227.633 151.348V171.958C227.633 173.95 227.882 175.239 228.38 175.825C229.201 176.85 230.636 177.51 232.687 177.803C233.537 177.978 233.961 178.96 233.961 180.747C233.961 181.977 233.8 182.812 233.478 183.252C233.185 183.662 232.716 183.911 232.072 183.999C231.427 184.087 230.783 184.248 230.138 184.482C229.435 184.717 228.908 185.068 228.556 185.537C227.941 186.299 227.633 187.705 227.633 189.756V204.17C227.633 211.494 225.436 215.815 221.041 217.134C219.577 217.573 217.863 217.793 215.9 217.793H208.693C207.873 217.793 207.462 216.753 207.462 214.673C207.462 212.622 207.873 211.597 208.693 211.597H212.384C215.226 211.597 217.101 210.967 218.009 209.707C218.712 208.74 219.064 206.792 219.064 203.862V190.679C219.064 185.64 221.701 182.344 226.974 180.791C221.701 179.473 219.064 176.235 219.064 171.079V151.611C219.064 148.682 218.771 146.777 218.185 145.898C217.101 144.316 215.094 143.525 212.165 143.525H208.693C207.873 143.525 207.462 142.471 207.462 140.361Z"
        fill="#8BB3D0"
      />
    </g>
    <defs>
      <filter
        color-interpolation-filters="sRGB"
        filterUnits="userSpaceOnUse"
        height="403.107"
        id="filter0_d"
        width="403.107"
        x="-24.7917"
        y="-20.7917"
      >
        <feFlood
          flood-opacity="0"
          result="BackgroundImageFix"
        />
        <feColorMatrix
          in="SourceAlpha"
          type="matrix"
          values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
        />
        <feOffset
          dy="4"
        />
        <feColorMatrix
          type="matrix"
          values="0 0 0 0 0.256007 0 0 0 0 0.587194 0 0 0 0 0.608333 0 0 0 0.59 0"
        />
        <feBlend
          in2="BackgroundImageFix"
          mode="normal"
          result="effect1_dropShadow"
        />
        <feBlend
          in="SourceGraphic"
          in2="effect1_dropShadow"
          mode="normal"
          result="shape"
        />
      </filter>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear"
        x1="206.91"
        x2="370.981"
        y1="54.7253"
        y2="252.819"
      >
        <stop
          stop-color="#E469B3"
        />
        <stop
          offset="0.536458"
          stop-color="#714CD8"
          stop-opacity="0.81"
        />
        <stop
          offset="1"
          stop-color="#20A4CE"
        />
      </linearGradient>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint1_linear"
        x1="203.901"
        x2="348.362"
        y1="73.7349"
        y2="212.865"
      >
        <stop
          stop-color="#2F5684"
        />
        <stop
          offset="1"
          stop-color="#1E3D63"
        />
      </linearGradient>
      <clipPath
        id="clip0"
      >
        <rect
          fill="white"
          height="353"
          width="353"
        />
      </clipPath>
    </defs>
  </svg>
</div>
`;

exports[`<CoditorLogo />  should render properly when name is passed 1`] = `
<div
  className="CoditorLogo inline-flex justify-center"
>
  <svg
    fill="none"
    height={100}
    viewBox="0 0 353 353"
    width={100}
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0)"
    >
      <g
        filter="url(#filter0_d)"
        id=""
      >
        <rect
          height="226"
          rx="57"
          stroke="url(#paint0_linear)"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="22"
          transform="rotate(53.2373 199.658 18.6041)"
          width="226"
          x="199.658"
          y="18.6041"
        />
      </g>
      <rect
        fill="url(#paint1_linear)"
        height="165.322"
        rx="35"
        transform="rotate(53.2373 193.443 61)"
        width="165.322"
        x="193.443"
        y="61"
      />
      <path
        d="M138.755 211.597C139.575 211.597 139.985 212.622 139.985 214.673C139.985 216.753 139.575 217.793 138.755 217.793H131.548C127.475 217.793 124.531 216.826 122.715 214.892C120.781 212.812 119.814 209.238 119.814 204.17V190.063C119.814 187.837 119.565 186.416 119.067 185.801C118.276 184.834 116.841 184.204 114.761 183.911C113.911 183.794 113.486 182.739 113.486 180.747C113.486 178.96 113.911 177.978 114.761 177.803C116.811 177.51 118.086 177.041 118.584 176.396C119.404 175.371 119.814 173.774 119.814 171.606V151.348C119.814 143.994 122.012 139.57 126.406 138.076C127.871 137.578 129.585 137.329 131.548 137.329H138.755C139.575 137.329 139.985 138.34 139.985 140.361C139.985 142.471 139.575 143.525 138.755 143.525H135.283C132.353 143.525 130.42 144.228 129.482 145.635C128.75 146.719 128.384 148.711 128.384 151.611V171.167C128.384 174.8 126.963 177.51 124.121 179.297C123.125 179.941 121.909 180.439 120.474 180.791C125.747 182.314 128.384 185.61 128.384 190.679V204.258C128.384 206.953 128.662 208.682 129.219 209.443C130.244 210.879 132.266 211.597 135.283 211.597H138.755Z"
        fill="#8BB3D0"
      />
      <path
        d="M190.181 169.277L187.632 171.211C185.084 168.047 181.802 166.465 177.789 166.465C174.302 166.465 171.314 167.71 168.824 170.2C166.363 172.661 165.132 175.635 165.132 179.121C165.132 182.607 166.363 185.596 168.824 188.086C171.314 190.547 174.302 191.777 177.789 191.777C179.752 191.777 181.583 191.352 183.282 190.503C185.01 189.653 186.475 188.481 187.676 186.987L190.137 189.053C188.673 190.898 186.856 192.349 184.688 193.403C182.52 194.429 180.22 194.941 177.789 194.941C173.424 194.941 169.688 193.403 166.583 190.327C163.507 187.222 161.968 183.486 161.968 179.121C161.968 174.756 163.507 171.035 166.583 167.959C169.688 164.853 173.424 163.301 177.789 163.301C180.25 163.301 182.564 163.843 184.732 164.927C186.9 165.981 188.716 167.432 190.181 169.277ZM195.103 165.322L192.598 167.256C190.841 165.029 188.658 163.286 186.05 162.026C183.472 160.766 180.718 160.137 177.789 160.137C172.545 160.137 168.062 161.997 164.341 165.718C160.65 169.438 158.804 173.906 158.804 179.121C158.804 184.336 160.65 188.804 164.341 192.524C168.062 196.245 172.545 198.105 177.789 198.105C180.718 198.105 183.487 197.475 186.094 196.216C188.702 194.927 190.884 193.169 192.642 190.942L195.059 192.964C193.008 195.542 190.474 197.578 187.457 199.072C184.439 200.537 181.216 201.269 177.789 201.269C173.775 201.269 170.069 200.288 166.671 198.325C163.272 196.333 160.577 193.638 158.585 190.239C156.622 186.841 155.64 183.135 155.64 179.121C155.64 175.107 156.622 171.401 158.585 168.003C160.577 164.604 163.272 161.924 166.671 159.961C170.069 157.969 173.775 156.973 177.789 156.973C181.216 156.973 184.439 157.72 187.457 159.214C190.504 160.679 193.052 162.715 195.103 165.322ZM200.069 161.367L197.608 163.345C195.235 160.386 192.32 158.057 188.863 156.357C185.406 154.658 181.715 153.808 177.789 153.808C174.361 153.808 171.08 154.482 167.945 155.83C164.84 157.148 162.144 158.95 159.859 161.235C157.603 163.491 155.801 166.186 154.454 169.321C153.135 172.427 152.476 175.693 152.476 179.121C152.476 182.549 153.135 185.83 154.454 188.965C155.801 192.07 157.603 194.766 159.859 197.051C162.144 199.307 164.84 201.108 167.945 202.456C171.08 203.774 174.361 204.433 177.789 204.433C181.715 204.433 185.406 203.584 188.863 201.885C192.32 200.185 195.235 197.856 197.608 194.897L199.981 196.963C197.315 200.273 194.049 202.881 190.181 204.785C186.314 206.66 182.183 207.598 177.789 207.598C173.922 207.598 170.23 206.85 166.715 205.356C163.199 203.833 160.167 201.811 157.618 199.292C155.098 196.743 153.077 193.711 151.553 190.195C150.059 186.68 149.312 182.988 149.312 179.121C149.312 175.254 150.059 171.562 151.553 168.047C153.077 164.531 155.098 161.514 157.618 158.994C160.167 156.445 163.199 154.424 166.715 152.93C170.23 151.406 173.922 150.644 177.789 150.644C182.213 150.644 186.358 151.597 190.225 153.501C194.122 155.405 197.403 158.027 200.069 161.367ZM204.991 157.412L202.442 159.302C199.483 155.64 195.85 152.754 191.544 150.644C187.266 148.535 182.681 147.48 177.789 147.48C173.511 147.48 169.41 148.315 165.484 149.985C161.588 151.655 158.218 153.911 155.377 156.753C152.564 159.565 150.323 162.934 148.653 166.86C146.983 170.757 146.148 174.844 146.148 179.121C146.148 183.398 146.983 187.5 148.653 191.426C150.323 195.322 152.564 198.691 155.377 201.533C158.218 204.346 161.588 206.587 165.484 208.257C169.41 209.927 173.511 210.762 177.789 210.762C182.681 210.762 187.266 209.707 191.544 207.598C195.85 205.488 199.483 202.602 202.442 198.94L204.947 200.874C201.695 204.917 197.696 208.11 192.95 210.454C188.233 212.768 183.179 213.926 177.789 213.926C173.072 213.926 168.56 213.003 164.254 211.157C159.976 209.311 156.285 206.836 153.179 203.73C150.074 200.625 147.598 196.933 145.753 192.656C143.907 188.349 142.984 183.838 142.984 179.121C142.984 174.404 143.907 169.907 145.753 165.63C147.598 161.323 150.074 157.617 153.179 154.512C156.285 151.406 159.976 148.931 164.254 147.085C168.56 145.239 173.072 144.316 177.789 144.316C183.179 144.316 188.248 145.488 192.994 147.832C197.74 150.146 201.739 153.34 204.991 157.412Z"
        fill="#E9B34B"
      />
      <path
        d="M207.462 140.361C207.462 138.34 207.873 137.329 208.693 137.329H216.076C219.972 137.329 222.843 138.369 224.689 140.449C226.623 142.617 227.604 146.25 227.633 151.348V171.958C227.633 173.95 227.882 175.239 228.38 175.825C229.201 176.85 230.636 177.51 232.687 177.803C233.537 177.978 233.961 178.96 233.961 180.747C233.961 181.977 233.8 182.812 233.478 183.252C233.185 183.662 232.716 183.911 232.072 183.999C231.427 184.087 230.783 184.248 230.138 184.482C229.435 184.717 228.908 185.068 228.556 185.537C227.941 186.299 227.633 187.705 227.633 189.756V204.17C227.633 211.494 225.436 215.815 221.041 217.134C219.577 217.573 217.863 217.793 215.9 217.793H208.693C207.873 217.793 207.462 216.753 207.462 214.673C207.462 212.622 207.873 211.597 208.693 211.597H212.384C215.226 211.597 217.101 210.967 218.009 209.707C218.712 208.74 219.064 206.792 219.064 203.862V190.679C219.064 185.64 221.701 182.344 226.974 180.791C221.701 179.473 219.064 176.235 219.064 171.079V151.611C219.064 148.682 218.771 146.777 218.185 145.898C217.101 144.316 215.094 143.525 212.165 143.525H208.693C207.873 143.525 207.462 142.471 207.462 140.361Z"
        fill="#8BB3D0"
      />
    </g>
    <defs>
      <filter
        color-interpolation-filters="sRGB"
        filterUnits="userSpaceOnUse"
        height="403.107"
        id="filter0_d"
        width="403.107"
        x="-24.7917"
        y="-20.7917"
      >
        <feFlood
          flood-opacity="0"
          result="BackgroundImageFix"
        />
        <feColorMatrix
          in="SourceAlpha"
          type="matrix"
          values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
        />
        <feOffset
          dy="4"
        />
        <feColorMatrix
          type="matrix"
          values="0 0 0 0 0.256007 0 0 0 0 0.587194 0 0 0 0 0.608333 0 0 0 0.59 0"
        />
        <feBlend
          in2="BackgroundImageFix"
          mode="normal"
          result="effect1_dropShadow"
        />
        <feBlend
          in="SourceGraphic"
          in2="effect1_dropShadow"
          mode="normal"
          result="shape"
        />
      </filter>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear"
        x1="206.91"
        x2="370.981"
        y1="54.7253"
        y2="252.819"
      >
        <stop
          stop-color="#E469B3"
        />
        <stop
          offset="0.536458"
          stop-color="#714CD8"
          stop-opacity="0.81"
        />
        <stop
          offset="1"
          stop-color="#20A4CE"
        />
      </linearGradient>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint1_linear"
        x1="203.901"
        x2="348.362"
        y1="73.7349"
        y2="212.865"
      >
        <stop
          stop-color="#2F5684"
        />
        <stop
          offset="1"
          stop-color="#1E3D63"
        />
      </linearGradient>
      <clipPath
        id="clip0"
      >
        <rect
          fill="white"
          height="353"
          width="353"
        />
      </clipPath>
    </defs>
  </svg>
  <span
    className="text-4xl mt-6 font-display text-indigo-900 font-extrabold"
  >
    Coditor
  </span>
</div>
`;

exports[`<CoditorLogo />  should render properly when size and animate property passed 1`] = `
<div
  className="CoditorLogo"
>
  <svg
    fill="none"
    height={200}
    viewBox="0 0 353 353"
    width={200}
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      clip-path="url(#clip0)"
    >
      <g
        filter="url(#filter0_d)"
        id="frame-rotate"
      >
        <rect
          height="226"
          rx="57"
          stroke="url(#paint0_linear)"
          stroke-linecap="round"
          stroke-linejoin="round"
          stroke-width="22"
          transform="rotate(53.2373 199.658 18.6041)"
          width="226"
          x="199.658"
          y="18.6041"
        />
      </g>
      <rect
        fill="url(#paint1_linear)"
        height="165.322"
        rx="35"
        transform="rotate(53.2373 193.443 61)"
        width="165.322"
        x="193.443"
        y="61"
      />
      <path
        d="M138.755 211.597C139.575 211.597 139.985 212.622 139.985 214.673C139.985 216.753 139.575 217.793 138.755 217.793H131.548C127.475 217.793 124.531 216.826 122.715 214.892C120.781 212.812 119.814 209.238 119.814 204.17V190.063C119.814 187.837 119.565 186.416 119.067 185.801C118.276 184.834 116.841 184.204 114.761 183.911C113.911 183.794 113.486 182.739 113.486 180.747C113.486 178.96 113.911 177.978 114.761 177.803C116.811 177.51 118.086 177.041 118.584 176.396C119.404 175.371 119.814 173.774 119.814 171.606V151.348C119.814 143.994 122.012 139.57 126.406 138.076C127.871 137.578 129.585 137.329 131.548 137.329H138.755C139.575 137.329 139.985 138.34 139.985 140.361C139.985 142.471 139.575 143.525 138.755 143.525H135.283C132.353 143.525 130.42 144.228 129.482 145.635C128.75 146.719 128.384 148.711 128.384 151.611V171.167C128.384 174.8 126.963 177.51 124.121 179.297C123.125 179.941 121.909 180.439 120.474 180.791C125.747 182.314 128.384 185.61 128.384 190.679V204.258C128.384 206.953 128.662 208.682 129.219 209.443C130.244 210.879 132.266 211.597 135.283 211.597H138.755Z"
        fill="#8BB3D0"
      />
      <path
        d="M190.181 169.277L187.632 171.211C185.084 168.047 181.802 166.465 177.789 166.465C174.302 166.465 171.314 167.71 168.824 170.2C166.363 172.661 165.132 175.635 165.132 179.121C165.132 182.607 166.363 185.596 168.824 188.086C171.314 190.547 174.302 191.777 177.789 191.777C179.752 191.777 181.583 191.352 183.282 190.503C185.01 189.653 186.475 188.481 187.676 186.987L190.137 189.053C188.673 190.898 186.856 192.349 184.688 193.403C182.52 194.429 180.22 194.941 177.789 194.941C173.424 194.941 169.688 193.403 166.583 190.327C163.507 187.222 161.968 183.486 161.968 179.121C161.968 174.756 163.507 171.035 166.583 167.959C169.688 164.853 173.424 163.301 177.789 163.301C180.25 163.301 182.564 163.843 184.732 164.927C186.9 165.981 188.716 167.432 190.181 169.277ZM195.103 165.322L192.598 167.256C190.841 165.029 188.658 163.286 186.05 162.026C183.472 160.766 180.718 160.137 177.789 160.137C172.545 160.137 168.062 161.997 164.341 165.718C160.65 169.438 158.804 173.906 158.804 179.121C158.804 184.336 160.65 188.804 164.341 192.524C168.062 196.245 172.545 198.105 177.789 198.105C180.718 198.105 183.487 197.475 186.094 196.216C188.702 194.927 190.884 193.169 192.642 190.942L195.059 192.964C193.008 195.542 190.474 197.578 187.457 199.072C184.439 200.537 181.216 201.269 177.789 201.269C173.775 201.269 170.069 200.288 166.671 198.325C163.272 196.333 160.577 193.638 158.585 190.239C156.622 186.841 155.64 183.135 155.64 179.121C155.64 175.107 156.622 171.401 158.585 168.003C160.577 164.604 163.272 161.924 166.671 159.961C170.069 157.969 173.775 156.973 177.789 156.973C181.216 156.973 184.439 157.72 187.457 159.214C190.504 160.679 193.052 162.715 195.103 165.322ZM200.069 161.367L197.608 163.345C195.235 160.386 192.32 158.057 188.863 156.357C185.406 154.658 181.715 153.808 177.789 153.808C174.361 153.808 171.08 154.482 167.945 155.83C164.84 157.148 162.144 158.95 159.859 161.235C157.603 163.491 155.801 166.186 154.454 169.321C153.135 172.427 152.476 175.693 152.476 179.121C152.476 182.549 153.135 185.83 154.454 188.965C155.801 192.07 157.603 194.766 159.859 197.051C162.144 199.307 164.84 201.108 167.945 202.456C171.08 203.774 174.361 204.433 177.789 204.433C181.715 204.433 185.406 203.584 188.863 201.885C192.32 200.185 195.235 197.856 197.608 194.897L199.981 196.963C197.315 200.273 194.049 202.881 190.181 204.785C186.314 206.66 182.183 207.598 177.789 207.598C173.922 207.598 170.23 206.85 166.715 205.356C163.199 203.833 160.167 201.811 157.618 199.292C155.098 196.743 153.077 193.711 151.553 190.195C150.059 186.68 149.312 182.988 149.312 179.121C149.312 175.254 150.059 171.562 151.553 168.047C153.077 164.531 155.098 161.514 157.618 158.994C160.167 156.445 163.199 154.424 166.715 152.93C170.23 151.406 173.922 150.644 177.789 150.644C182.213 150.644 186.358 151.597 190.225 153.501C194.122 155.405 197.403 158.027 200.069 161.367ZM204.991 157.412L202.442 159.302C199.483 155.64 195.85 152.754 191.544 150.644C187.266 148.535 182.681 147.48 177.789 147.48C173.511 147.48 169.41 148.315 165.484 149.985C161.588 151.655 158.218 153.911 155.377 156.753C152.564 159.565 150.323 162.934 148.653 166.86C146.983 170.757 146.148 174.844 146.148 179.121C146.148 183.398 146.983 187.5 148.653 191.426C150.323 195.322 152.564 198.691 155.377 201.533C158.218 204.346 161.588 206.587 165.484 208.257C169.41 209.927 173.511 210.762 177.789 210.762C182.681 210.762 187.266 209.707 191.544 207.598C195.85 205.488 199.483 202.602 202.442 198.94L204.947 200.874C201.695 204.917 197.696 208.11 192.95 210.454C188.233 212.768 183.179 213.926 177.789 213.926C173.072 213.926 168.56 213.003 164.254 211.157C159.976 209.311 156.285 206.836 153.179 203.73C150.074 200.625 147.598 196.933 145.753 192.656C143.907 188.349 142.984 183.838 142.984 179.121C142.984 174.404 143.907 169.907 145.753 165.63C147.598 161.323 150.074 157.617 153.179 154.512C156.285 151.406 159.976 148.931 164.254 147.085C168.56 145.239 173.072 144.316 177.789 144.316C183.179 144.316 188.248 145.488 192.994 147.832C197.74 150.146 201.739 153.34 204.991 157.412Z"
        fill="#E9B34B"
      />
      <path
        d="M207.462 140.361C207.462 138.34 207.873 137.329 208.693 137.329H216.076C219.972 137.329 222.843 138.369 224.689 140.449C226.623 142.617 227.604 146.25 227.633 151.348V171.958C227.633 173.95 227.882 175.239 228.38 175.825C229.201 176.85 230.636 177.51 232.687 177.803C233.537 177.978 233.961 178.96 233.961 180.747C233.961 181.977 233.8 182.812 233.478 183.252C233.185 183.662 232.716 183.911 232.072 183.999C231.427 184.087 230.783 184.248 230.138 184.482C229.435 184.717 228.908 185.068 228.556 185.537C227.941 186.299 227.633 187.705 227.633 189.756V204.17C227.633 211.494 225.436 215.815 221.041 217.134C219.577 217.573 217.863 217.793 215.9 217.793H208.693C207.873 217.793 207.462 216.753 207.462 214.673C207.462 212.622 207.873 211.597 208.693 211.597H212.384C215.226 211.597 217.101 210.967 218.009 209.707C218.712 208.74 219.064 206.792 219.064 203.862V190.679C219.064 185.64 221.701 182.344 226.974 180.791C221.701 179.473 219.064 176.235 219.064 171.079V151.611C219.064 148.682 218.771 146.777 218.185 145.898C217.101 144.316 215.094 143.525 212.165 143.525H208.693C207.873 143.525 207.462 142.471 207.462 140.361Z"
        fill="#8BB3D0"
      />
    </g>
    <defs>
      <filter
        color-interpolation-filters="sRGB"
        filterUnits="userSpaceOnUse"
        height="403.107"
        id="filter0_d"
        width="403.107"
        x="-24.7917"
        y="-20.7917"
      >
        <feFlood
          flood-opacity="0"
          result="BackgroundImageFix"
        />
        <feColorMatrix
          in="SourceAlpha"
          type="matrix"
          values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
        />
        <feOffset
          dy="4"
        />
        <feGaussianBlur
          stdDeviation="14"
        />
        <feColorMatrix
          type="matrix"
          values="0 0 0 0 0.256007 0 0 0 0 0.587194 0 0 0 0 0.608333 0 0 0 0.59 0"
        />
        <feBlend
          in2="BackgroundImageFix"
          mode="normal"
          result="effect1_dropShadow"
        />
        <feBlend
          in="SourceGraphic"
          in2="effect1_dropShadow"
          mode="normal"
          result="shape"
        />
      </filter>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint0_linear"
        x1="206.91"
        x2="370.981"
        y1="54.7253"
        y2="252.819"
      >
        <stop
          stop-color="#E469B3"
        />
        <stop
          offset="0.536458"
          stop-color="#714CD8"
          stop-opacity="0.81"
        />
        <stop
          offset="1"
          stop-color="#20A4CE"
        />
      </linearGradient>
      <linearGradient
        gradientUnits="userSpaceOnUse"
        id="paint1_linear"
        x1="203.901"
        x2="348.362"
        y1="73.7349"
        y2="212.865"
      >
        <stop
          stop-color="#2F5684"
        />
        <stop
          offset="1"
          stop-color="#1E3D63"
        />
      </linearGradient>
      <clipPath
        id="clip0"
      >
        <rect
          fill="white"
          height="353"
          width="353"
        />
      </clipPath>
    </defs>
  </svg>
</div>
`;
